packages/components/pie-assistive-text/src/assistive-text.scss

```diff
@@ -1 +1,27 @@
 @use '@justeattakeaway/pie-css/scss' as p;
+
+.c-assistiveText {
+    --assistive-text-color: var(--dt-color-content-default);
+
+    padding-block-start: var(--dt-spacing-a);
+    color: var(--assistive-text-color);
+
+    &[variant='success'] {
+        --assistive-text-color: var(--dt-color-content-positive);
+    }
+
+    &[variant='error'] {
+        --assistive-text-color: var(--dt-color-content-error);
+    }
+}
+
+
+.c-assistiveText-content {
```

Comment:
<img width="376" alt="Screenshot 2024-02-06 at 13 20 26" src="https://github.com/justeattakeaway/pie/assets/16766185/aa660cb8-ca7f-4208-8c59-6e0e4968c41a">

Looks great! Just want to double check, do we want to remove the default margin applied to the `p` tag?

Reply:
<img width="256" alt="Screenshot 2024-02-06 at 13 21 29" src="https://github.com/justeattakeaway/pie/assets/16766185/caa8b724-41b9-46ff-8915-550f17ecabfd">

Also, are you happy that it meets the whitespace requirements outlined in the design? I only ask because I have trouble understanding them sometimes :D